libr/core/core.c

```diff
@@ -1276,7 +1277,11 @@ R_API int r_core_init(RCore *core) {
 
 	r_core_config_init (core);
 
+	ut64 diff, prev = r_sys_now();
 	r_core_loadlibs_init (core);
+	diff = r_sys_now() - prev;
+	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
```

Comment:
measure plugin load time: r_core_loadlibs

measure file bin parsing: r_core_file_open*   .. see which ones are calling r_bin_load()


> On 17 Mar 2016, at 20:50, Alexandru Caciulescu <notifications@github.com> wrote:
> 
> In libr/core/core.c <https://github.com/radare/radare2/pull/4332#discussion_r56566759>:
> 
> >  	r_core_loadlibs_init (core);
> > +	diff = r_sys_now() - prev;
> > +	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
> @Maijin <https://github.com/Maijin> My bad, I noticed when I tried ?T I get a valid output instead of command not implemented or smth similar so I assumed it was taken, I see now it ignores the T for the moment.
> The file loading time question still stands btw.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/radare/radare2/pull/4332/files/c4861bba3c8f0dcffe22d93dcf27d8d31087fbaa#r56566759>